resources/main.css

```diff
@@ -547,6 +560,9 @@ section#details .non-standard-params h2 {
 #non-standard-params-table tbody td {
     padding: 0.1em 0.3em;
 }
+#non-standard-params-table tbody td:nth-child(2) {
+    color: var(--highlight);
```

Comment:
This red isn't very contrasted:
![image](https://github.com/user-attachments/assets/e344c5b1-37cc-41b9-9c74-83962167d623)

Wanna make it lighter?